privacy/ccpa/policy.go

```diff
@@ -44,7 +55,17 @@ func ReadFromRequestWrapper(req *openrtb_ext.RequestWrapper) (Policy, error) {
 }
 
 func ReadFromRequest(req *openrtb2.BidRequest) (Policy, error) {
```

Comment:
Can we add some tests for this function now that it is no longer a one-liner?

Reply:
Will need to construct some GPP strings to enable that.